Apps/frontend/src/components/Waves.svelte

```diff
@@ -91,9 +92,9 @@
       }
       xLast[channelIndex] = xNew;
 
-      let sweep = $timeSweep[channelIndex] / 5.0 - 1.0;
-      let fac = sweep < 0 ? MIN_SWEEP : MAX_SWEEP;
-      let delta = fac * sweep + 1.0;
+      // time sweep
+      let sweep = $timeSweep[channelIndex] / 5.0 - 1.0;// in [-1,1]
```

Comment:
Please improve documentation about calculations. Why do we need to do / 5.0 -1

Reply:
Documentation for this can be found in the Wiki